composer/core/data_spec.py

```diff
@@ -260,7 +261,14 @@ def _default_get_num_samples_in_batch(self, batch: Batch) -> int:
                     Please use a DataSpec and specify `get_num_samples_in_batch`."""))
 
     def _default_get_num_tokens_in_batch(self, batch: Batch) -> int:
-        del batch  # unused
+        # first try HuggingFace-style input dicts
```

Comment:
```suggestion
        # First try HuggingFace-style input dicts
```
nit. Is this in style guide?

Reply:
idk, i guess my habit is just not capitalizing lol